### PR TITLE
Fix #1440: Add unit test for AssertionChallengeConverter

### LINKED ISSUE #1440: Add unit test for AssertionChallengeConverter

### PR DESCRIPTION
**Pull request by jnpsk**
`AssertionChallengeConverter`'s methods were all made static and test of the methods were added.

### CODE REVIEW

File: powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverter.java [hunk 1]
```diff
@@ -113,7 +115,6 @@ public AssertionChallenge convertAssertionChallengeFromOperationDetail(Operation
                 final List<String> transports = (List<String>) ad.getExtras().get("transports");
                 final String aaguid = (String) ad.getExtras().get("aaguid");
 
-                // Obtain credential ID, append data to credential ID if the authenticator is a Wultra authenticator that supports visual challenge.
```
Comment (banterCZ) [inline powerauth-fido2/src/main/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverter.java:116]: Thanks, the comment was really outdated. If you touched the line, `credentialId` could by final.

File: powerauth-fido2/src/test/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverterTest.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+        assertNotNull(assertionChallenge.getAllowCredentials());
+        final AllowCredentials allowCredential = assertionChallenge.getAllowCredentials().get(0);
+        assertArrayEquals("credential-1".getBytes(), allowCredential.getCredentialId());
+        assertEquals("hybrid", allowCredential.getTransports().get(0));
+        assertEquals("public-key", allowCredential.getType());
+    }
+
+    @Test
+    void testConvertAssertionChallengeFromOperationDetail_withWultraAuthenticatorDetail() {
```
Comment (banterCZ) [inline powerauth-fido2/src/test/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverterTest.java:131]: Have you considered a test case for multiple Wultra authentication details?

**Review by banterCZ — DISMISSED** [review]
Perfect, thanks for the test.

File: powerauth-fido2/src/test/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverterTest.java (hunk 2 above)
Comment (jnpsk) [reply powerauth-fido2/src/test/java/com/wultra/powerauth/fido2/rest/model/converter/AssertionChallengeConverterTest.java:131]: good idea, I added the test

**Pull request merged**